spec/support/acceptance/nssdb.rb

```diff
@@ -0,0 +1,65 @@
+begin
+  require 'serverspec'
+rescue LoadError
+  # Not using acceptance tests
+else
+  module Serverspec
+    module Type
+      class Nssdb < Command
+        def certificates
+          # TODO: parse the output
+          command_result
+        end
+
+        private
+
+        def command
+          "certutil -L -d #{@name}"
+        end
+
+        def command_result
+          @command_result ||= @runner.run_command(command)
+        end
```

Comment:
Does this need a `command_result` method? What is unique to `Nssdb` that can't be inherited from `Command`?

Reply:
Because the default implementation calls `@name`. I've submitted https://github.com/mizzy/serverspec/pull/616 which would make this easier.